endpoints/horses/index.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable no-continue */
+/* eslint-disable no-mixed-operators */
+/* eslint-disable no-plusplus */
```

Comment:
`eslint` fails locally for me on this rule in this file but I have no idea why 🤔